CHANGES.md

```diff
@@ -1,3 +1,15 @@
+# grmtools 0.13.3 (2023-09-20)
+
+* Explicitly disallow lines in an lrlex input file rules from starting with a
+  space (POSiX lex treats these as verbatim code, but lrlex has no such
+  support).
```

Comment:
Ok to squash pending those changes above which you feel are worth doing.

Reply:
Spelling fixed in 9ddb293.

I did wonder whether to classify this as a breaking change or not. I eventually came down on the side of "we never said that users could could have leading space", but I'm prepared to be convinced otherwise.

Reply:
I don't feel terribly strongly about it enough to argue the case that it should be classified as a breaking change.
More was just wondering if it was worth mentioning that there was a change of behavior in the release notes.

Reply:
Good point. Fixed in 4705bab.